AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -536,6 +536,7 @@ autoblog.com#@#.ad
 !
 !
 !
+agrihaber04.com,afyonhaberturk.com,nehaberankara.com,mansetburdur.com,bolgesellig.com,telgrafgazetesi.com,burokratika.com,balikesirartihaber.com,ajansbalikligol.com,karar67.com,gazetemalatya.com,vansesigazetesi.com,kocaelisabah.com,antalyadanhaberler.com,aydinyeniufuk.com.tr,yerelvanhaber.com,a2teker.com,kibrishakikat.com,izmirtime35.com,sancakplus.com,marasfisilti.com,burdurgazetesi.com,zeytinburnuhaber.org,seffafbelediyecilik.com,denizli20haber.com,akdenizdeyeniyuzyil.net,ulakci.com,egegundem.com.tr,isdunyasindakadin.com,haberimizvar.net#%#//scriptlet('abort-on-property-read', 'isAdBlockActive')
```

Comment:
This rule is ok, but we also have platforms that don't support advanced rules. They are called content blockers.
Could you check these sites with this rule `@@/advert.js$domain=`? After `$domain=` you should add the list of sites.
I have checked only two of them and it seems ok.

Reply:
Do you want it replaced with `@@/advert.js$domain=agrihaber04.com|afyonhaberturk.com|nehaberankara.com|mansetburdur.com|bolgesellig.com|telgrafgazetesi.com|burokratika.com|balikesirartihaber.com|ajansbalikligol.com|karar67.com|gazetemalatya.com|vansesigazetesi.com|kocaelisabah.com|antalyadanhaberler.com|aydinyeniufuk.com.tr|yerelvanhaber.com|a2teker.com|kibrishakikat.com|izmirtime35.com|sancakplus.com|marasfisilti.com|burdurgazetesi.com|zeytinburnuhaber.org|seffafbelediyecilik.com|denizli20haber.com|akdenizdeyeniyuzyil.net|ulakci.com|egegundem.com.tr|isdunyasindakadin.com|haberimizvar.net`?

Reply:
Not replace - also add for content blockers.
1. Don't remove the scriptlet rule.
2. Check an exception rule on these sites.
3. If it's ok, you can add it to the Annoyances - antiadblock.txt - look at the end of the file - you will see the section. Add the exception rule here.
```
! Common safari_cb_affinity(general)
!#safari_cb_affinity(general)
```

Reply:
With the other rule, on `https://www.aydinyeniufuk.com.tr` there is something on the right side.

Reply:
Ok, fixed.